libs/ts/config-types/tests/config-decoding.spec.ts

```diff
@@ -0,0 +1,173 @@
+import * as path from 'path';
+import { describe, expect, test } from 'vitest';
+
+import { selectDirectory } from '@blocksense/base-utils/fs';
+import { configDir, configDirs, configFiles } from '@blocksense/base-utils/env';
+
+import {
+  decodeFeedsConfig,
+  decodeNewFeedsConfig,
+  FeedsConfigSchema,
+  NewFeedsConfigSchema,
+} from '../src/data-feeds-config';
+
+import {
+  ChainlinkCompatibilityConfigSchema,
+  decodeChainlinkCompatibilityConfig,
+} from '../src/chainlink-compatibility/types';
+
+import {
+  DeploymentConfigSchemaV1,
+  decodeDeploymentConfigV1,
+  DeploymentConfigSchemaV2,
+  decodeDeploymentConfigV2,
+} from '../src/evm-contracts-deployment';
+
+describe('Configuration files decoding', async () => {
+  const { readJSON, decodeJSON } = selectDirectory(configDir);
+
+  // Names of the configuration files
+  const feedConfigV1 = path.parse(configFiles.feeds_config_v1).name;
```

Comment:
`configFiles` should be a mapping containing both the path and the schema used for decoding. That way reading a config file will always use the correct schema. And also this test could be written to iterate over all files.